packages/ecc-client-ga4gh-tes/src/components/tes-run/README.md

```diff
@@ -46,6 +46,15 @@ This attribute indicates the current `state` or `status` of the `task`. In the g
 | :-------- | :------- | :-------------------------------- | :-------|
 |`state`|`string`|**Required**. State of the task|null|
 
+#### admin 
+
+This attribute is used to allow user to be able to be able interact with the task at admin level, ie delete (as of now). 
+
+| Parameter | Type     | Description                       | Default |
```

Comment:
add a column for your parameters to indicate if they are necessary or not. Do that for all your components too

Reply:
In all the API references, if an attr is required, I have mentioned it in bold since I have followed that throughout my components; changing that wouldn't make sense.

Should I still do it?

Reply:
okay, if you think it's unnecessary then its fine